www/index.js

```diff
@@ -4,11 +4,7 @@ import './css/intro.css';
 import './css/appstatus.css';
 import './css/main.recent.css';
 import './css/main.diary.css';
-import './manual_lib/enketo/css/formhub.css';
-import './manual_lib/enketo/css/formhub-print.css';
 import './manual_lib/fontawesome/css/all.min.css';
-
-import 'leaflet/dist/leaflet-src.js';
```

Comment:
I understand that we don't need this any more since the leaflet hack is no longer required. However, can't we also then remove the leaflet.css file (next line)?

Reply:
No, we still have to get the Leaflet styles from somewhere.
It's not coming from `manual_lib`; this is the official Leaflet package from npm (latest version). We don't have anything for Leaflet in `manual_lib` anymore.

But one good thing to do sometime is refactor all uses of the `L` global into modular imports. Leaflet 2.0, when it is released, is not going to provide the `L` global.

Reply:
So webpack automatically includes js through tree shaking but not css?

Reply:
Here's my best understanding of it. Webpack, when it analyzes our project, is only able to build its dependency graph based on ES `import` statements or CommonJS `require()` statements (in our project I've been sticking to the former).

In the case of Leaflet, we don't use either of these; we rely on a global variable `L`. I have to specifically provide this `L` in `webpack.config.js` https://github.com/e-mission/e-mission-phone/pull/980/files#diff-1fb26bc12ac780c7ad7325730ed09fc4c2c3d757c276c3dacc44bfe20faf166f

Then the CSS has to be separately imported. I could do it here or I could do it somewhere else, like `styles.css`. But it doesn't happen automatically.